src/init.cpp

```diff
@@ -1558,6 +1558,8 @@ bool AppInitMain(NodeContext& node, interfaces::BlockAndHeaderTipInfo* tip_info)
                                      PeerManager::Options{
                                          .banman = node.banman.get(),
                                          .ignore_incoming_txs = ignores_incoming_txs,
+                                         .reconcile_txs =
+                                             args.GetBoolArg("-txreconciliation", DEFAULT_TXRECONCILIATION_ENABLE),
```

Comment:
Could reading from the args be moved to their own `ApplyArgsManOptions` function like in various `node/*_args.cpp` files? Then the test code wouldn't have to repeat the logic here.

Reply:
Done on the latest push